src/views/home/Followers.vue

```diff
@@ -1,20 +1,40 @@
 <template>
   <div>
-    <h2>{{ title }}</h2>
+    <v-list>
+      <v-list-item-group color="primary">
+        <FollowingActor
+          v-for="(following, index) in getFollowing"
+          :key="index"
+          :actor="following"
+        />
+      </v-list-item-group>
+    </v-list>
   </div>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from "vue-property-decorator";
+import { namespace } from "vuex-class";
+import { ActivityObject, Link } from "activitypub-objects";
+import FollowingActor from "@/components/following/FollowingActor.vue";
```

Comment:
I always group the imports according to external and internal

Reply:
So what would be the correct sorting? Don't I have the external ones first and the internal one second?

Reply:
Badly explained. I separate the external and internal with a blank line in between

```
import { Component, Vue } from "vue-property-decorator";
import { namespace } from "vuex-class";
import { ActivityObject, Link } from "activitypub-objects";

import FollowingActor from "@/components/following/FollowingActor.vue";
```

Reply:
Ah, got it. Misunderstood that "grouping" thing^^